### PR TITLE
PP 195 User is able to see Payment Summary on Payment Details page

### PR DESCRIPTION
**Pull request by acceleanu**
https://payments-platform.atlassian.net/browse/PP-195
User is able to see Payment Summary on Payment Details page with what they're paying for
- persist the payment description field in the charge 
- refactoring

### CODE REVIEW

**Comment by acceleanu** [comment]
rebuild

File: src/test/java/uk/gov/pay/connector/it/dao/ChargeDaoITest.java [hunk 1]
```diff
@@ -31,31 +31,30 @@
...
     @Before
     public void setUp() throws Exception {
         chargeDao = new ChargeDao(app.getJdbi());
         app.getDatabaseTestHelper().addGatewayAccount(gatewayAccountId, "test_account");
+
+        chargeId = chargeDao.saveNewCharge(newCharge(amount));
     }
 
     @Test
     public void insertANewChargeAndReturnTheId() throws Exception {
```
Comment (zadsas) [inline src/test/java/uk/gov/pay/connector/it/dao/ChargeDaoITest.java]: Isn't this test going to fail if you move it to the bottom of the file? It relies on being executed the first one.

Comment (acceleanu) [inline src/test/java/uk/gov/pay/connector/it/dao/ChargeDaoITest.java]: No

**Comment by PauloPortugal** [comment]
rebuild

**Pull request merged**